CHANGELOG.mdx

```diff
@@ -16,6 +16,14 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.0.7] - 2020-11-24
+                       
+
+### Bugfixes
+- [#5974](https://github.com/rasahq/rasa/issues/5974): `ActionRestart` will now trigger `ActionSessionStart` as a followup action.
+- [#7317](https://github.com/rasahq/rasa/issues/7317): Fixed Rasa Open Source not being able to fetch models from certain URLs.
```

Comment:
@erohmensing can we elaborate here that this fixes an issue with Rasa X that was introduced in version 2.0.3?

Reply:
Yes! Had a window open for this lol

Reply:
lol thanks :D

Reply:
```suggestion
- [#7317](https://github.com/rasahq/rasa/issues/7317): Fixed Rasa Open Source not being able to fetch models from certain URLs.

  This addresses an issue introduced in 2.0.3 where `rasa-production` could not use the models from `rasa-x` in Rasa X server mode.
```